new/vcloud-manager-blueprint.yaml

```diff
@@ -0,0 +1,876 @@
+tosca_definitions_version: cloudify_dsl_1_2
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/3.3m7/types.yaml
+  - http://www.getcloudify.org/spec/fabric-plugin/1.3m7/plugin.yaml
+  - types/manager-types.yaml
+  - https://raw.githubusercontent.com/cloudify-cosmo/tosca-vcloud-plugin/1.3m7/plugin.yaml
+
+
+
+
+inputs:
+
+  vcloud_username:
+    type: string
+    description: >
+      User login for vcloud air
+
+  vcloud_password:
+    type: string
+    default: ''
+    description: >
+      User password for vcloud air - for login by name + password
+
+  vcloud_token:
+    type: string
+    default: ''
+    description: >
+      User token for vcloud air - for login by name + token
+
+  catalog:
+    type: string
+    default: Public Catalog
+    description: >
+      Name of catalog, can be 'Public Catalog'
+
+  template:
+    type: string
+    default: CentOS-7-x86_64-1406
+    description: >
+      Name of template from catalog,
+      can be 'CentOS-7-x86_64-1406'
+
+  management_network_name:
+    type: string
+    description: >
+      Name common network that can be used for nodes
+
+  edge_gateway:
+    type: string
+    default: gateway
+    description: >
+      For 'ondemand' service type, the value of edge_gateway
+      is always 'gateway'
+
+  server_name:
+    type: string
+    default: centosmanager
+    description: >
+      Human readible name for server
+
+  vcloud_url:
+    type: string
+    default: 'vca.vmware.com'
+    description: >
+      Vcloud url
+
+  vcloud_service:
+    type: string
+    default: ''
+    description: >
+      Vcloud service
+
+  vcloud_service_type:
+    type: string
+    default: 'subscription'
+    description: >
+      Type of service: subscription, ondemand, vcd, private
+
+  vcloud_api_version:
+    type: string
+    default: '5.7'
+    description: >
+      Version of api, for now 5.6
+
+  vcloud_vdc:
+    type: string
+    description: >
+      Virtual data center name
+
+  vcloud_org:
+    type: string
+    description: >
+      Organization name
+
+  vcloud_instance:
+    type: string
+    default: ''
+    description: >
+      Only required for ondemand service type, instance uuid
+
+  ssh_port:
+    type: string
+    default: 22
+    description: >
+      SSH port that can be used for installed node
+
+  network_use_existing:
+    type: boolean
+    default: false
+    description: >
+      Use existed network
+
+  ssl_verify:
+    type: boolean
+    default: true
+    description: >
+        ssl check for connections to private services
+        (disable self-signed certificates)
+
+  user_public_key:
+    type: string
+    description: >
+      Public part of your key
+
+  management_port_ip_allocation_mode:
+    type: string
+    default: dhcp
+    description: >
+      Ip allocation type for case when you doesn't set public ip
```

Comment:
'IP allocation type for cases when you don't set a public ip for a node and instead want to auto-allocate it.'